javascript/react/src/components/verbs/__tests__/Workflows.stories.tsx

```diff
@@ -9,11 +9,11 @@ import binning from './specs/binning.json'
 import categorical from './specs/categorical.json'
 import dropdown from './specs/dropdown.json'
 import everyOperation from './specs/every-operation.json'
-import multistepBinarize from './specs/multistep-binarize.json'
-import onehotunhot from './specs/onehot-unhot.json'
+import multiStepBinarize from './specs/multi-step-binarize.json'
```

Comment:
did these filenames get changed to match?

Reply:
All stories should work, which will verify

Reply:
These have been reverted to their original forms